package.json

```diff
@@ -488,54 +488,51 @@
             }
           }
         },
-        "rust.rls": {
+        "rust.rls.executable": {
           "default": null,
```

Comment:
Before the pull request `"rust.rls.executable"` was `"rls"` by default, but it didn't work (I didn't know about that).
The reason it didn't work is described in #341.